apps/voting/test/contracts/voting.js

```diff
@@ -114,6 +116,19 @@ contract('Voting App', ([root, holder1, holder2, holder20, holder29, holder51, n
             await assertRevert(voting.changeMinAcceptQuorumPct(neededSupport.plus(1)))
         })
 
+        it('can change the early execution setting', async () => {
+            assert.isTrue(await voting.isEarlyExecutionAllowed(), 'early execution should be allowed by default')
+
+            const receipt = await voting.changeEarlyExecution(false)
+            assertAmountOfEvents(receipt, 'ChangeEarlyExecution')
+            assertEvent(receipt, 'ChangeEarlyExecution', { allowed: false })
+
+            assert.isFalse(await voting.isEarlyExecutionAllowed(), 'early execution should be disallowed')
+        })
+
+        it('fails trying to set the same early execution setting', async () => {
+            await assertRevert(voting.changeEarlyExecution(true), 'VOTING_SAME_EARLY_EXECUTION')
```

Comment:
We should also add a test to check that the Voting app is correctly configured after initialization (e.g. has the right support / quorum, but also its default values for early execution and overrule window).

Reply:
isn't that being checked [here](https://github.com/aragon/aragon-apps/pull/886/files#diff-5af58e0b9d17bd3d4f22ad98562608bdR120)?

Reply:
will move it to a separate test to make it clearer